doc/classes/ORMMaterial3D.xml

```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ORMMaterial3D" inherits="BaseMaterial3D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		PBR Material that can be applied to 3D objects, can use an ORM texture.
```

Comment:
May wish to say: "Physically based rendering (PBR) Material that can be applied to 3D objects, can use an ORM texture."

New user-developers may not not know what PBR means and cannot guess from context like ORM can be as defined in the full description.